pennylane/ops/op_math/decompositions/two_qubit_unitary.py

```diff
@@ -532,7 +532,7 @@ def _decomposition_3_cnots(U, wires):
     D_ops = one_qubit_decomposition(D, wires[1])
 
     # Return the full decomposition
-    return C_ops + D_ops + interior_decomp + A_ops + B_ops
+    return C_ops + D_ops + interior_decomp + A_ops + B_ops + [qml.GlobalPhase(-alpha / 2)]
```

Comment:
There are multiple potential decomposition pathways, including `_decomposition_3_cnots`, `_decomposition_2_cnots`, `_decomposition_1_cnots` and `_decomposition_0_cnots`. Maybe we should add the same thing to the rest of them?

Reply:
The only one that mentions that does not have the global phase is the 3 cnots. Anyway now it fails a test that uses 2_cnots without modifying anything  🤔 (`/pennylane/tests/ops/op_math/test_decompositions.py::test_two_qubit_decomposition_special_case_discontinuity
`)I'm looking at it